recipes/protobuf-c/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
test_v1_package tests are no longer necessary, that was removed some days ago

Reply:
Thanks. Removed. It might be good to remove it from the template [here](https://github.com/gegles/conan-center-index/tree/master/docs/package_templates/cmake_package/all/test_v1_package) then.